sys/include/net/conn/udp.h

```diff
@@ -93,6 +96,8 @@ int conn_udp_getlocaladdr(conn_udp_t *conn, void *addr, uint16_t *port);
  *
  * @note    Function may block.
  *
+ * @note    This function needs to be called from the same function as @ref conn_udp_create.
```

Comment:
Yeah, but the `conn` can be used from any other function in that thread (e.g. if `conn` is global or handed to other function via parameter).